### PR TITLE
chore(ci): update helm chart version

### PR DESCRIPTION
**Pull request by pchandra19**

### CODE REVIEW

**Review by tiagolobocastro — CHANGES_REQUESTED** [review]
This is not necessary, once we tag it, GHA should upgrade the version.

File: chart/values.yaml [hunk 1]
```diff
@@ -52,7 +52,7 @@ base:
     containers:
       - name: etcd-probe
         image: busybox:latest
-        command: ['sh', '-c', 'trap "exit 1" TERM; until nc -vzw 5 {{ .Release.Name }}-etcd {{ .Values.etcd.service.port }}; do date; echo "Waiting for etcd..."; sleep 1; done;']
+        command: ['sh', '-c', 'trap "exit 1" TERM; until nc -vzw 5 {{ .Release.Name }}-etcd {{.Values.etcd.service.port}}; do date; echo "Waiting for etcd..."; sleep 1; done;']
```
Comment (avishnu) [inline chart/values.yaml:55]: Is the space not needed?

Comment (pchandra19) [reply chart/values.yaml:55]: No Vishnu, the space causes lint issues.

Comment (niladrih) [reply chart/values.yaml:55]: Fixing this helps us with picking the mayastor-extensions chart as-is without having to overwrite some things. The openebs/charts linter fails with these spaces. Typically the values yaml wasn't meant for templates, so doesn't have template support as much as the template directory stuff.

**Comment by pchandra19** [comment]
Closing this PR as its not required. We have instead fixed the GHA , thanks to @tiagolobocastro.

**Pull request closed without merging**